webclient/projekt/filters.py

```diff
@@ -665,6 +679,18 @@ def filtr_dokumenty_ident(self, queryset, name, value):
             | Q(casti_dokumentu__dokument__ident_cely__icontains=value)
         ).distinct()
 
+    def filter_pristupnost(self, queryset, name, value):
```

Comment:
@pesikj Mohu se mýlit, ale přijde mi, že ta funkce nereflektuje princip, jak se stanovuje přístupnost projektu. To je popsáno zde: https://github.com/ARUP-CAS/aiscr-webamcr/issues/2353#issue-2521927554

Takto to najde spíš přístupnosti podřazených záznamů, ale ne správně definovanou přístupnost projektu jako takového...

Reply:
Upraveno, aby to více odpovídalo logice `pristupnost_pom`.